src/_pytest/deprecated.py

```diff
@@ -29,3 +29,7 @@
     "--result-log is deprecated and scheduled for removal in pytest 6.0.\n"
     "See https://docs.pytest.org/en/latest/deprecations.html#result-log-result-log for more information."
 )
+
+SCOPE_AS_NON_KEYWORD = PytestDeprecationWarning(
+    "Fixture scope shall be passed as a keyword argument."
```

Comment:
```suggestion
    "Passing the scope to pytest.fixture() as a positional argument is deprecated - pass it as a keyword argument scope= instead."
```

(with proper wrapping...)